src/components/sync-connected-sites.tsx

```diff
@@ -314,12 +317,15 @@ const SyncConnectedSitesSection = ( {
 														'push'
 													) }
 													placement="top-start"
-													disabled={ isOffline }
+													disabled={ isOffline || isAnySitePulling || isAnySitePushing }
```

Comment:
I considered moving this to a separate constant such as `syncDisabled` but so far it seemed easier to read when it said actual conditions. Happy to move it if it is easier to read.